cumulus-tf/main.tf

```
@@ -19,10 +19,10 @@ module "cumulus" {
   deploy_to_ngap = true
 
   vpc_id            = var.vpc_id
-  lambda_subnet_ids = var.subnet_ids
+  lambda_subnet_ids = var.lambda_subnet_ids
 
   ecs_cluster_instance_image_id   = var.ecs_cluster_instance_image_id
-  ecs_cluster_instance_subnet_ids = var.subnet_ids
+  ecs_cluster_instance_subnet_ids = var.ecs_cluster_instance_subnet_ids
```

Might I suggest making this an optional override? Like:

```suggestion
  ecs_cluster_instance_subnet_ids = var.ecs_cluster_instance_subnet_ids ? var.ecs_cluster_instance_subnet_ids : var.subnet_ids
```